addons/mail/views/mail_message_views.xml

```diff
@@ -139,44 +139,31 @@
                     <field name="id"/>
                     <field name="mimetype"/>
                     <field name="type"/>
-                    <field name="create_uid"/>
-                    <field name="url"/>
-                    <field name="create_date"/>
-                    <field name="name"/>
                     <templates>
                         <t t-name="kanban-menu">
                             <a t-attf-href="/web/content/ir.attachment/#{record.id.raw_value}/datas?download=true" download="" class="dropdown-item">Download</a>
                             <a t-if="widget.deletable" role="menuitem" type="delete" class="dropdown-item">Delete</a>
                         </t>
-                        <t t-name="kanban-box">
-                            <div class="oe_kanban_global_area oe_kanban_global_click o_kanban_attachment">
-                                <div class="o_kanban_image">
-                                    <div class="o_kanban_image_wrapper">
-                                        <t t-set="webimage" t-value="new RegExp('image.*(gif|jpeg|jpg|png|webp)').test(record.mimetype.value)"/>
-                                        <div t-if="record.type.raw_value == 'url'" class="o_url_image fa fa-link fa-3x text-muted" aria-label="Image is a link"/>
-                                        <img t-elif="webimage" t-attf-src="/web/image/#{record.id.raw_value}" width="100" height="100" alt="Document" class="o_attachment_image"/>
-                                        <div t-else="!webimage" class="o_image o_image_thumbnail" t-att-data-mimetype="record.mimetype.value"/>
-                                    </div>
+                        <t t-name="kanban-card" class="o_kanban_attachment flex-row">
+                            <div class="o_kanban_image">
```

Comment:
isn't this a classical aside/main layout? Why do we need those custom (that previously where standard) classnames (o_kanban_image, o_kanban_details, o_kanban_details_wrapper) ?

Reply:
As you can observe in the `kanban_view.scss`, the height and width of the kanban are primarily determined by `$o-kanban-attachment-image-size`, which corresponds to the largest attachment uploaded. If we don't keep these classes, the height of the kanban will vary across different rows. Therefore, we need to use the CSS classes to maintain consistent dimensions.